documentai/snippets/quickstart_sample_test.py

```diff
@@ -16,25 +16,42 @@
 # flake8: noqa
```

Comment:
Don't disable flake8 entirely without a comment explaining why it was necessary.

Reply:
I tried to disable just one error (ANN201 for return values, not sure if there's a way to do this for the whole file instead of adding it to one line) because I'm returning a value outside the displayed sample and I don't want people looking at the docs to get confused.